--- /MTEngine/Games/c64/Emulators/atari800/AtariInterface/AtariWrapper.mm	2021-04-19 09:46:41.644914544 +0200
+++ /MTEngine/Games/c64/Emulators/atari800/AtariInterface/AtariWrapper.mm.new	2021-04-19 09:54:04.031947062 +0200
@@ -147,7 +147,7 @@
 
 #if !defined(RUN_ATARI)
 // TODO: workaround for now to allow compile without Atari emu, remove me
-volatile UWORD CPU_regPC;
+//volatile UWORD CPU_regPC;
 #endif
 
 int atrd_debug_pause_check(int allowRestore)
